Source/SwiftLintFramework/Models/Location.swift

```diff
@@ -26,7 +26,18 @@ public struct Location: CustomStringConvertible, Comparable {
         self.character = character
     }
 
-    public init(file: File, offset: Int) {
+    public init(file: File, byteOffset offset: Int) {
+        self.file = file.path
+        if let lineAndCharacter = file.contents.lineAndCharacterForByteOffset(offset) {
+            line = lineAndCharacter.line
+            character = lineAndCharacter.character
```

Comment:
I found that `character` is distance of utf16index, not `String.characters.count`.
But, it seems Xcode count columns by utf16index instead of characters on displaying annotation as following:
<img width="794" alt="screenshot 2015-12-25 12 27 48" src="https://cloud.githubusercontent.com/assets/33430/11999503/fa27d4ec-ab02-11e5-9ad7-9074ad447bd8.png">
So, I stop fix that.